rules/rbe_repo.bzl

```diff
@@ -504,56 +804,78 @@ def rbe_autoconfig(
     Args:
       name: Name of the rbe_autoconfig repository target.
       base_container_digest: Optional. If the container to use for the RBE build
-          extends from the rbe-ubuntu16-04 image, you can pass the digest
+          extends from the container defined in the toolchain_config_suite_spec
+          (by default, the rbe-ubuntu16-04 image), you can pass the digest
           (sha256 sum) of the base container using this attr.
-          The rule will enable use of checked-in configs if possible.
+          The rule will try to use of checked-in configs, if possible.
       bazel_version: The version of Bazel to use to generate toolchain configs.
           `Use only (major, minor, patch), e.g., '0.20.0'. Default is "local"
           which means the same version of Bazel that is currently running will
           be used. If local is a non release version, rbe_autoconfig will fallback
           to using the latest release version (see _BAZEL_VERSION_FALLBACK).
+          Note, if configs are not found for a patch version, rule will attempt
```

Comment:
Does Bazel guarantee patch versions are backwards compatible?

Reply:
Yes they do, patch versions should only be to fix regressions.